### PR TITLE
Observable Schema Refactor

### PR DESCRIPTION
**Pull request by Padraic-O-Mhuiris**

### CODE REVIEW

File: src/components/CDPDisplay/Presentation.js [hunk 1]
```diff
@@ -48,9 +48,9 @@ export default function({ cdp, showSidebar, account, network, cdpOwner }) {
     .getContractAddress('CDP_MANAGER');
 
   // Testing:
-  const { dsProxy } = useObservable('dsProxy', account?.address);
-  const { getVaults } = useObservable('getVaults', manager, dsProxy);
-  const { ilkPrice } = useObservable('ilkPrice', 'ETH-A');
+  const { dsProxy } = watch.dsProxy(account?.address);
+  const { getVaults } = watch.getVaults(manager, dsProxy);
+  const { ilkPrice } = watch.ilkPrice('ETH-A');
```
Comment (b-pmcg) [inline src/components/CDPDisplay/Presentation.js]: Dude, this is slick! Importing the schemas from the plugin and adding into the watcher in MakerProvider works really nice too.

**Comment by levity** [comment]
looks like CircleCI is failing because some yalc configuration was accidentally checked in

**Comment by b-pmcg** [comment]
> looks like CircleCI is failing because some yalc configuration was accidentally checked in

Yeah, we haven't published a beta of the SDK and the plugin yet, and since we're all using yalc while things are still in development it seemed more convenient to leave it in instead of re-linking with every pull. 

I believe the SDK & plugin are stable enough that we could publish a beta though, that would make Circle happy.

**Comment by Padraic-O-Mhuiris** [comment]
> This sort of bug is going to stop happening completely when we merge the new multicall stuff, right?

```
#!/bin/bash -eo pipefail
yarn install
yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
error Package "" refers to a non-existing file '"/home/circleci/project/.yalc/@makerdao/dai"'.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exited with code exit status 1
```

Yes, publishing a new package will allow us to remove the yalc imports

**Comment by levity** [comment]
I was referring to the set max bug 🙂

**Comment by Padraic-O-Mhuiris** [comment]
@levity Checks will pass after deploying and adding new versions for mcd-plugin. Dai.js core may need a publish also, Michael has been working on some validation stuff with it. We're really close now so aiming for next week to have it finally merged.

**Pull request merged**